mmdet/models/backbones/resnet.py

```diff
@@ -29,13 +30,19 @@ def __init__(self,
                  dilation=1,
                  downsample=None,
                  style='pytorch',
-                 with_cp=False):
+                 with_cp=False,
+                 normalize=dict(type='BN')):
         super(BasicBlock, self).__init__()
         self.conv1 = conv3x3(inplanes, planes, stride, dilation)
-        self.bn1 = nn.BatchNorm2d(planes)
+
+        # build_norm_layer return: (norm_name, norm_layer)
+        self.norm1, norm1 = build_norm_layer(normalize, planes, postfix=1)
```

Comment:
The name of `self.norm1` may be misleading, `self.norm1_name` instead?

Reply:
How about using self.norm1_ instead of self.norm1_name for short?